src/features/workflow/state/workflowState.ts

```diff
@@ -39,28 +34,20 @@ export const SET_RUN_MODE = 'SET_RUN_MODE'
 // of the workflow without having to have a working api
 export const TEMP_SET_WORKFLOW_EVENTS = 'TEMP_SET_WORKFLOW_EVENTS'
 
-export const selectLatestDryRun = (state: RootState): Run | undefined => {
+export const selectLatestDryRunId = (state: RootState): string | undefined => {
   if (state.workflow.lastDryRunID) {
-    return NewRunFromEventLog(state.workflow.lastDryRunID, state.workflow.events)
+    return state.workflow.lastDryRunID
   }
   return undefined
 }
 
-export const selectLatestRun = (state: RootState): Run | undefined => {
+export const selectLatestRunId = (state: RootState): string | undefined => {
   if (state.workflow.lastRunID) {
-    return NewRunFromEventLog(state.workflow.lastRunID, state.workflow.events)
+    return state.workflow.lastRunID
   }
   return undefined
 }
```

Comment:
let's refactor these to return a `string` only. If there is no `lastDryRunID` or `lastRunID`, return the empty string.